modules/configuration/php/NDB_Form_configuration.class.inc

```diff
@@ -106,6 +106,30 @@ class NDB_Form_Configuration extends NDB_Form
         $this->tpl_data['sandbox']         = $config->getSetting("sandbox");
 
     }
+
+    function subproject() {
+        $factory = NDB_Factory::singleton();
+        $config = $factory->config();
+
+        $subprojectList = Utility::getSubprojectList();
+        $subprojects = array();
+        foreach($subprojectList as $subprojectID => $title) {
+            $subprojects[$subprojectID] = $config->getSubprojectSettings($subprojectID);
+        }
+        $this->tpl_data['subprojects'] = $subprojects;
+
+        $this->tpl_data['useEDCOptions']
```

Comment:
I don't think so, it should be either true or false. null is pretty meaningless, and not having an empty option forces them to choose one.